### ISSUE TITLE
Unable to convert large books

### ISSUE THREAD

**Issue by josteinaj** [nlbdev/pipeline#114]
*from Karis google sheet:*

- book 500259 (2051KB): fails
- book 558237 (1310KB): fails
- book 500156 (1091KB): fails
- book 56527616 (1030KB): ok

**Comment by bertfrees** [comment]
Where can I find these books? On the test server?

**EDIT**: Never mind, thought I was assigned.

**Comment by josteinaj** [comment]
Yeah it's not uploaded anywhere yet I think. I assigned you first but changed it to me, so that I can create xprocspec tests for it before handing it over to you.

**Comment by josteinaj** [comment]
So I've tested the three books that are failing, and the problem seems to be about tables. Two of the books have the same problem as in #121. And a third book gives a 'Failed to solve table' error that I've created a separate issue for here: #129.

I've created 8 tests for books with simple structures in different sizes: 10 kB, 100 kB, 500 kB, 1 MB, 1.5MB, 2MB, 5MB and 10MB. All tests together run in about 20 minutes. So I actually don't think there's an error when trying to convert big books (although big books might take a few more minutes to convert).

Let's keep this issue open until #121 and #129 is fixed though, so that we can test again after that.
